src/components/bulkEdits/bulkEdit.css

```diff
@@ -1,23 +1,19 @@
-.select-width{
+.select-width {
     width: 240px;
 }
 
-.updated-container{
+.updated-container {
     height: 'calc(100vh - 100px)';
     min-height: 0;
     display: grid;
     grid-template-rows: 48px 1fr;
 }
 
-.updated-container--sample{
+.updated-container--sample {
     height: calc(100vh - 150px);
     align-items: flex-start;
 }
 
-.brdr-btm{
```

Comment:
is it intentional?

Reply:
Yes , in base.scss file we have border-bottom already. so replaced this with border-bottom